automates/text_reading/src/main/resources/org/clulab/aske_automates/grammars/functions.yml

```diff
@@ -316,12 +318,155 @@ rules:
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "They suggested that the main factors controlling α included leaf area index (LAI), vapor pressure deficit (VPD) and soil moisture content (θ)."
+    pattern: |
+      trigger = (?<= [lemma = "factor"]|[lemma = "variable"]) [word = "controlling"]|[word = "influencing"]
+      input:Phrase+ = <acl <nsubj dobj | <amod <nsubj
+      output:Phrase = dobj | <amod
+
+  - name: obtained_with
+    label: Function
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "For simulations conducted, ET with fm = 0.0 and 1.0 was obtained with the input variables, such as Rn and LAI, parameterized as the same as those of fm = 0.5, which might be imprecise and compromise the results."
+    pattern: |
+      trigger = [word = "obtained"] (?= [word = "with"])
+      input:Identifier+ = nmod_with nmod_such_as
+      output:Phrase = <ccomp nsubj | nsubjpass
+
+  - name: mean_or_average_output
+    label: Function
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "the mean α obtained from the 26 available days was 1.25 ± 0.03."
+    pattern: |
+      trigger = [word = /mean|average/]
+      input:Phrase+ = <compound | <amod
+      output:Phrase = <nsubj | <amod nsubj
+
+  # function rules allowing partial extraction
+
+  - name: compute_output
+    label: Function
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "It also computes the root water uptake of each soil layer."
+    pattern: |
+      trigger = [lemma = "compute"]
+      input:Phrase* = dobj? nmod_from conj_and? | xcomp dobj | advcl_based_on | nsubj:xsubj | nmod_for nmod_on nmod_of | <amod nmod_of
+      output:Phrase? = nsubjpass conj_or? | dobj | <amod <nmod_from nsubjpass
+
+  - name: required_as_input
+    label: Function
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "The daily weather values as well as all soil properties and current soil water content, by layer, are required as input."
+    pattern: |
+      trigger = [lemma = "require"] (?= [word = "as"] ([word = "input"]|[word = "inputs"]))
+      input:Phrase+ = nsubjpass conj_and? | nmod_as
+      output:Phrase? = nmod_for
+
+  - name: average_fragment
+    label: Function
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "Each day’s observations were averaged separately;"
+    pattern: |
+      trigger = [lemma = average]
+      input:Phrase+ = nsubjpass
+
+  - name: the_resulting_values
+    label: Function
+    priority: ${priority}
+    type: dependency
+    action: ${action}
+    example: "the resulting average values of cpT were plotted against those of Lq, and a line was fitted by eye to find the best value of cpΔT/LΔ."
+    pattern: |
+      trigger = [word = "resulting"]
+      output:Phrase = <amod
+  # fixme: when trigger and output overlaps, extraction does not happen.
```

Comment:
The desired output here is `average values of cpT`. (This extraction was needed as a part of the cross-sentence function) However, this is not extracted because the trigger `resulting` and part of the output `average values` are captured as a single phrase, and thus overlap.